core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -1061,8 +1062,9 @@ public synchronized ZookeeperLockChecker getTServerLockChecker() {
       // because that client could be closed, and its ZooSession also closed
       // this needs to be fixed; TODO https://github.com/apache/accumulo/issues/2301
       var zk = info.getZooKeeperSupplier(ZookeeperLockChecker.class.getSimpleName()).get();
-      this.zkLockChecker =
-          new ZookeeperLockChecker(new ZooCache(zk), getZooKeeperRoot(), getServerPaths());
+      this.zkLockChecker = new ZookeeperLockChecker(
+          new ZooCache(zk, Optional.empty(), ZooUtil.getRoot(getInstanceID())), getZooKeeperRoot(),
```

Comment:
Why use both `ZooUtil.getRoot()` and `getZooKeeperRoot()`?

Reply:
good comment - I was just porting over changes from my previous PR and didn't catch that.

Reply:
Fixed in 8d0a9cd

Reply:
I fixed the conflict here with #5253 , but I don't think I got the formatting right.